Lesson_9_Modeling/Lesson_9_Modeling.ipynb

```diff
@@ -0,0 +1,4606 @@
+{
```

Comment:
<p>This question is a little vague. I'm not really sure what you mean. Maybe provide some hints?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>